base/common/src/test/java/com/netscape/certsrv/profile/ProfileInputTest.java

```diff
@@ -12,7 +14,8 @@
 
     @Before
     public void setUpBefore() {
-        before.addAttribute(new ProfileAttribute("sn_uid", "user", null));
+        before.addAttribute(new ProfileAttribute("sn_uid", "user", new Descriptor(IDescriptor.CHOICE, "true,false,-", "-", "CMS_PROFILE_CRITICAL")));
+        before.addConfigAttribute(new ProfileAttribute("sn_abc", "configattr", new Descriptor(IDescriptor.CHOICE, "true,false,-", "-", "CMS_PROFILE_CRITICAL")));
```

Comment:
I think it is easier to read if you define the descriptor up top like [this](https://github.com/dogtagpki/pki/pull/3705/files#diff-615af683a66402875f0a87b4738867b0a5f9b85d3eb01c92db9242b5c6d3ad01R23-R27), that way you only have to define it once too.

Reply:
Agreed, will push a change